new-lamassu-admin/src/pages/Authentication/Login.js

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, Grid } from '@material-ui/core'
+import React from 'react'
+
+import styles from './Login.styles'
+import LoginCard from './LoginCard'
+
+const useStyles = makeStyles(styles)
+
+const Login = () => {
+  const classes = useStyles()
+
+  return (
+    <>
+      <Grid
+        container
+        spacing={0}
+        direction="column"
+        alignItems="center"
+        justify="center"
+        style={{ minHeight: '100vh' }}
```

Comment:
Is there a reason to have this here? Could it be in css?

Reply:
You are right, going to correct it.